scripts/createFromTemplate.ps1

```diff
@@ -187,6 +187,11 @@ Get-ChildItem -Force -File -Recurse * | ForEach-Object {
                 ForEach-Object {
                     $_ -replace "__home__",$env:HOME
                 } | Set-Content $a
+                ( Get-Content $a ) |
+                ForEach-Object {
+                    # Get just the last string of the path, which is the name of the folder of the template
```

Comment:
Let's keep the code and comments where possible with 80 columns. The code style is not specified, I know, but I'll open an issue for that.